babel-watch.js

```diff
@@ -247,18 +246,21 @@ function restartApp() {
     }
   }
 
-  // Support for --debug option
   const runnerExecArgv = process.execArgv.slice();
-  if (program.debug) {
-    runnerExecArgv.push('--debug=' + program.debug);
-  }
-  // Support for --inspect option
-  if (program.inspect) {
-    runnerExecArgv.push('--inspect');
-  }
-  // Support for --debug-brk
-  if(program.debugBrk) {
-    runnerExecArgv.push('--debug-brk');
+  const unknownOptions = program.parseOptions(program.normalize(process.argv.slice(2))).unknown;
+
+  if (unknownOptions.length > 0) {
+    for (let i = 0, len = unknownOptions.length; i < len; ++i) {
+      // If this unknown option is a '--' option and the next option looks like it might
+      // be an argument for this option, we format it as --option=argument and pass it on.
+      if (unknownOptions[i].indexOf('--') === 0 &&
+          unknownOptions[i + 1] &&
+          unknownOptions[i + 1].indexOf('-') !== 0) {
+        runnerExecArgv.push(unknownOptions[i] + '=' + unknownOptions[++i]);
```

Comment:
Use space instead of `=`:
```
$ touch test.js
$ node --require=./test.js
node: bad option: --require=./test.js
$ node --require ./test.js
>
```